src/RiakClient.Tests.Live/RiakClientTests.cs

```diff
@@ -111,7 +115,7 @@ public void AsyncDeleteMultipleIsSuccessful()
             var oneObjectId = one.ToRiakObjectId();
             var twoObjectId = two.ToRiakObjectId();
 
-            var list = new List<RiakObjectId> { oneObjectId, twoObjectId };
+            var list = new List<RiakObjectId> {oneObjectId, twoObjectId};
```

Comment:
We should standardize our formatting here otherwise we'll just keep changing code back and forth. I'm using the default of including spaces and I can't seem to find the setting in "Tools -> Options -> Text Editor -> C# -> Formatting" anywhere. Maybe it's R# specific?